docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md

```diff
@@ -23,7 +23,7 @@ Client and services can communicate through many different types of communicatio
 
 The first axis is defining if the protocol is synchronous or asynchronous:
```

Comment:
nit unrelated to your PR for lines 24 and 30: is defining --> defines

Reply:
Fixed, and line 30 too.